src/main/java/uk/ac/ebi/embl/flatfile/writer/embl/EmblReducedFlatFileWriter.java

```diff
@@ -0,0 +1,104 @@
+package uk.ac.ebi.embl.flatfile.writer.embl;
+
+import java.io.IOException;
+import java.io.Writer;
+import java.util.HashSet;
+
+import uk.ac.ebi.embl.api.entry.Entry;
+import uk.ac.ebi.embl.api.entry.feature.Feature;
+import uk.ac.ebi.embl.api.entry.feature.SourceFeature;
+import uk.ac.ebi.embl.api.entry.reference.Reference;
+import uk.ac.ebi.embl.flatfile.EmblTag;
+import uk.ac.ebi.embl.flatfile.writer.EntryWriter;
+import uk.ac.ebi.embl.flatfile.writer.WrapType;
+import uk.ac.ebi.ena.taxonomy.taxon.Taxon;
+
+/** Reduced flat file writer.
+ */
+public class EmblReducedFlatFileWriter extends EntryWriter {
+
+    public EmblReducedFlatFileWriter(Entry entry) {
+        super(entry);
+        wrapType = WrapType.EMBL_WRAP;
+    }
+
+    private boolean excludeSource = true;
```

Comment:
- Suggestion to make 'excludeSource' private final to prevent it from being changed (no business reason exists for this):
```
private final boolean excludeSource = true; 
```
- The excludeSource setter is not required and neither is the getter i think.